src/translator/byte_array_util.cpp

```
@@ -0,0 +1,41 @@
+#include "byte_array_util.h"
+#include <stdlib.h>
+#include <fstream>
+#include <iostream>
+
+namespace marian {
+namespace bergamot {
+
+MemoryGift loadFileToMemory(const std::string& path, bool isModelFile){
+  std::ifstream is (path, std::ifstream::binary);
+  ABORT_IF(is.bad(), "Failed opening file stream: {}", path);
+  uint64_t length = 0; // Determine the length of file in bytes
+  is.seekg(0, is.end);
+  length = is.tellg();
+  is.seekg(0, is.beg);
```

What if file isn't seekable?  This will fail silently.